setup.py

```diff
@@ -25,6 +25,7 @@
     "paramiko>=1.18.5,<2",
     "psycopg2>=2.7.7",
     "requests>=2.21",
+    "six>=1.12.0",
```

Comment:
Added because CI failed without it. Please see https://travis-ci.org/OpenUpSA/wazimap/builds/641827861